### PR TITLE
Update protocol.md

### PR DESCRIPTION
**Pull request by RenaKunisaki**
Add more commands and info

### CODE REVIEW

File: protocol.md [hunk 1]
```diff
@@ -123,20 +140,23 @@ Arguments:
...
+
 Request body: NUL-terminated command, at most 255 bytes including terminator.
+
 Response body: standard output of the command.
 
 The command is split on spaces and then passed to `execvp`. In order to see
 standard error, use variables and globbing, use a command such as:
 
-    sh -c eval\t"$*"</dev/null\t2>&1 -- echo $PATH
+    `sh -c eval\t"$*"</dev/null\t2>&1 -- echo $PATH`
```
Comment (Lekensteyn) [inline protocol.md]: why the backticks? For markdown four spaces are sufficient to mark it as code

**Comment by culots** [comment]
> Purpose of `ONRS` and `AATD` are unknown. Both seem to reboot normally. Probably one is meant to enter fastboot?

AATD should reboot the device into the bootable FTM or FACTORY partition, to the MiniOS instance living there. Maybe to start the All Auto Test (AAT) automatically, without requesting user intervention first, which it usually does. That is, if the kernel supports 'aat_enter' in its restart.c.

Used to:

File: protocol.md (hunk 1 above)
Comment (RenaKunisaki) [reply protocol.md]: Atom editor didn't recognize it as code and caused formatting issues in the following text.

Comment (Lekensteyn) [reply protocol.md]: (File a bug with the Atom editor 😉 )

Does it still show correctly in Atom if you remove the initial four spaces here?

**Comment by Lekensteyn** [comment]
I've reformatted the changes to fit within 80 columns, fixed the typo reported by @shinobisoft and other smaller changes ("eg" -> "for example").

I have not incorporated @culots's feedback, if you have a good guess about it, please propose a PR. Thanks everyone!

File: protocol.md (hunk 1 above)
Comment (shinobisoft) [reply protocol.md]: Found another typo... Line 133. Same typo I mentioned earlier, just in a different place...

**Pull request closed without merging**